packages/overlay/src/telemetry/components/events/AICopyButton.tsx

```diff
@@ -1,5 +1,6 @@
 import { captureException } from "@sentry/core";
-import { formatEventOutput, processErrorEvent } from "@spotlightjs/sidecar/format";
+import { processErrorEvent } from "@spotlightjs/sidecar/formatters/human/errors";
```

Comment:
I'd name this formatter `markdown` or `md` since we're gonna replace the human one soon?